src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -71,7 +71,7 @@
                 new DefaultIdentifierTranslator(internalSession);
         final ListMultimap<String, String> headers = ArrayListMultimap.create();
 
-        LOGGER.debug("Adding WebAC Link Header for Resource: {}", resource);
+        LOGGER.debug("Adding WebAC Link Header for Resource: {}", resource.getPath());
```

Comment:
Just as a side-note, we don't really have a good method to call here. `getPath` is really a JCR inheritance anyway. We can't constrain `toString`, that's impl territory. Maybe some kind of `getLabel` or something is appropriate in `kernel-api`.